src/wallet/wallet.cpp

```diff
@@ -1415,7 +1415,6 @@ void CWallet::BlockConnected(const CBlock& block, const std::vector<CTransaction
     // the notification that the conflicted transaction was evicted.
 
     for (const CTransactionRef& ptx : vtxConflicted) {
-        SyncTransaction(ptx, CWalletTx::TxState::CONFLICTED, {} /* block hash */, 0 /* position in block */);
```

Comment:
In commit "Remove SyncTransaction for conflicted txn in CWallet::BlockConnected" (0e3212ae85c7cf1eb6a8d937ac67d6822f624175)

I don't think I understand this change. Commit description suggests why it is ok, but I don't understand what motivates it. Is it just cleanup? Is it maybe a minor bugfix, or needed for a future change?

Reply:
Updated commit message. I think this redundant, as conflicts tagging logic is triggered by connection of conflicting transaction. AFAIK, it's not a bugfix but a cleanup. And as it was same issue than major commit, I bundled them together.

Reply:
Extended commit message, we keep the loop because set of conflicted txn isn't same as txn included in a block.